fastestimator/op/numpyop/univariate/onehot.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, Callable, Dict, Iterable, List, Optional, Union
+
+import numpy as np
+
+from fastestimator.op import NumpyOp
+
+
+class Onehot(NumpyOp):
+    """ Transform the label integer to one-hot-encoding
+        Ref: https://towardsdatascience.com/label-smoothing-making-model-robust-to-incorrect-labels-2fae037ffbd0
+
+    Args:
+        num_classes: Total number of classes.
+        label_smoothing: Smoothing factor, after smoothing class output is: 1 - label_smoothing + label_smoothing
+            / num_classes, the other class output is: label_smoothing / num_classes
+        inputs: Input key(s) of labels to be onehot encoded
+        outputs: Output key(s) of labels
+        mode: What execution mode (train, eval, None) to apply this operation
+    """
+    def __init__(self,
+                 num_classes: int,
+                 label_smoothing: Optional[float] = None,
```

Comment:
instead of making this an optional float why not just default to 0?

Reply:
good idea, done